nautobot/ui/src/components/Layout.js

```diff
@@ -23,6 +55,35 @@ export default function Layout({ children }) {
         document.location.reload(true);
     }
 
+    function changeFeedbackInputValue(e) {
+        setFeedbackFormState((state) => ({
+            ...state,
+            [e.target.name]: e.target.value,
+        }));
+    }
+
+    function handleLeaveFeedbackSubmit(e) {
+        e.preventDefault();
+        const formData = {
+            user_agent: navigator.userAgent,
+            view_name: window.location.pathname,
+            nautobot_version: packageJson.version,
```

Comment:
Looks like package.json still has `"version": "2.0.0-alpha.1"` which is wrong at this point. We should probably add a unit test asserting that it's always set correctly to match pyproject.toml?